src/migrations/20240118093611-missing-primary-keys.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+        ALTER TABLE project_stats ADD PRIMARY KEY (project);
+        ALTER TABLE api_token_project ADD PRIMARY KEY (secret);
+        ALTER TABLE role_permission ADD COLUMN id SERIAL PRIMARY KEY;
```

Comment:
No good column to add, so creating id column.

Reply:
Can we have a combined key? role_id + permission

Reply:
Nope, there is also environment column. So it would have to be triple column primary key.

Reply:
And no after checking, we can not use it, because environment is nullable and so is permission. We can not make environment not nullable, because root roles have no environment. So adding id seems only solution. @gastonfournier

Reply:
ok, I appreciate you checking this!

Reply:
Will it populate past rows?

Reply:
Yes, serial will auto populate it

Reply:
> Yes, serial will auto populate it

TIL!